.github/workflows/registry-docs-pr-based.yml

```diff
@@ -0,0 +1,179 @@
+on:
+  workflow_call:
+    inputs:
+      providerFqn:
+        description: "Provider FQN to convert, e.g. hashicorp/aws"
+        required: true
+        type: string
+      additionalProviderFqns:
+        description: "Additional provider FQNs to convert, e.g. 'hashicorp/azurerm,hashicorp/google'"
```

Comment:
```suggestion
        description: "Additional provider FQNs that are required to convert because they are used in examples, e.g. 'hashicorp/azurerm,hashicorp/google'"
```